### PR TITLE
Use lock to prevent multiple jobs run simultaneously

### PR DESCRIPTION
**Pull request by karniv00l**

### CODE REVIEW

File: src/Command/ProxySyncReleasesCommand.php [hunk 1]
```diff
@@ -39,6 +43,11 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $lock = $this->lockFactory->createLock('packagist_releases_feed');
+        if (!$lock->acquire()) {
+            return 0;
+        }
+
```
Comment (karniv00l) [inline src/Command/ProxySyncReleasesCommand.php:50]: lock is released automatically on instance destruction

**Comment by akondas** [comment]
we need to add settings for `codecov/patch` like for `codecov/project`

**Comment by akondas** [comment]
```
coverage:
    status:
        project:
            default:
                informational: true
        path:
            default:
                informational: true
```

File: src/Command/ProxySyncReleasesCommand.php [hunk 2]
```diff
@@ -59,6 +77,7 @@ private function syncPackages(\SimpleXMLElement $feed): void
         }
 
         foreach ($feed->channel->item as $item) {
+            $this->lock->refresh();
```
Comment (akondas) [inline src/Command/ProxySyncReleasesCommand.php:80]: I think `refresh` can happen in `if (isset($syncedPackages[$name])) {` condition

Comment (karniv00l) [reply src/Command/ProxySyncReleasesCommand.php]: done

**Pull request merged**